guice/src/main/java/com/flipkart/gjex/grpc/service/GrpcServer.java

```diff
@@ -62,7 +66,12 @@ public GrpcServer(GJEXConfiguration configuration,
 			info("Creating GrpcServer listening on port : " + port);
 			this.port = configuration.getGrpc().getPort();
 		}
-		this.grpcServerBuilder = ServerBuilder.forPort(this.port);
+
+		if (configuration.getGrpc().getMaxMessageSize() > 0) {
+			info("Creating GrpcServer with maximum message size allowed : " + maxMessageSize);
```

Comment:
here maxMessageSize is still the default value.

Reply:
ok i will change